pkgs/by-name/as/assetto-server/package.nix

```
@@ -0,0 +1,30 @@
+{ fetchFromGitHub
+, buildDotnetModule
+, lib
+, dotnetCorePackages
+}:
+
+buildDotnetModule rec {
+  pname = "AssettoServer";
+  version = "0.0.54";
+
+  src = fetchFromGitHub {
+    owner = "compujuckel";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-ax8xBpIEcBCSg9dSI7osC2cZFq+PrIzDUm7UpDVvAPI=";
+  };
+  dotnet-sdk = dotnetCorePackages.sdk_8_0;
+  dotnet-runtime = dotnetCorePackages.runtime_8_0;
+  selfContainedBuild = true;
+
+  projectFile = "AssettoServer/AssettoServer.csproj";
+  nugetDeps = ./deps.nix;
+
```

```suggestion

passthru.updateScript = nix-update-script { };

```

So we get [automatic updates](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#automatic-package-updates)